monitoring-guidelines.md

```diff
@@ -55,13 +55,19 @@ When creating a KubeVirt alert rule, please follow the [OpenShift Alerting Consi
 In addition to the [OpeShift Style Guide](https://github.com/openshift/enhancements/blob/master/enhancements/monitoring/alerting-consistency.md#style-guide) the KubeVirt alerts MUST include:
 1. `kubernetes_operator_part_of` label indicating the operator name. Value should be set to `kubevirt`.
 2. `kubernetes_operator_component` label indicating the value of the sub operator name.
-
+3. `operator_health_impact` label indicating how the alert impacts the operator's functionality. 
+This label differs from `severity`, as severity can be tied to workload issues (e.g VMIs), while  `operator_health_impact` refers only to the impact the issue has on the operator itself.
```

Comment:
> This label differs from `severity`, as severity can be tied to workload issues (e.g VMIs), while  `operator_health_impact` refers only to the impact the issue has on the operator itself.

For example, An alert may have a "warning" severity, when talking on the impact on the cluster health, but have a "critical" impact on the operator health. Also, When an alert is tied to a specific workload it can have a "warning" severity, but no impact on the operator health.

Reply:
Should be an addition to this sentence, or a replacement?